app/views/admin/statistics_announcements/show/_main.html.erb

```diff
@@ -0,0 +1,137 @@
+          items: [
+            { field: "Type", value: @statistics_announcement.display_type },
+            { field: "Organisations", value: @statistics_announcement.organisations.map(&:name).to_sentence },
+          ],
+        }
+      %>
+    </div>
+
+    <div>
+      <%= render "components/inset_prompt", {
+        error: true,
+        description: "Statistics release cancelled - #{@statistics_announcement.cancellation_reason}.",
+      } if @statistics_announcement.cancelled? %>
+    </div>
+  </section>
+
+  <section class="app-view-statistics-announcements-summary__section">
+    <div>
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Release and delivery date",
+        heading_level: 2,
+        font_size: "l",
+        margin_bottom: 3,
+      } %>
+      <p class="govuk-body">
+        <%= link_to("Change dates",
+                    new_admin_statistics_announcement_change_path(@statistics_announcement),
+                    class: "govuk-link",
+                    data: {
+                      module: "gem-track-click",
+                      "track-category": "button-clicked",
+                      "track-action": "#{@statistics_announcement.model_name.singular.dasherize}-button",
+                      "track-label": "Change dates",
+                    }) %>
+      </p>
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Release date",
+        heading_level: 3,
+        font_size: "m",
+        margin_bottom: 2,
+      } %>
+      <% if @statistics_announcement.cancelled? %>
+       <p>
+         <del class="govuk-body"><%= @statistics_announcement.release_date.to_fs(:date_with_time) %></del>
+         <span class="govuk-body"> (cancelled)</span>
+       </p>
+      <% else %>
+        <p class="govuk-body"> <%= @statistics_announcement.release_date.to_fs(:date_with_time) %> </p>
+      <% end %>
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Delivery date",
+        heading_level: 3,
+        font_size: "m",
+        margin_bottom: 2,
+      } %>
+      <p class="govuk-body">
+        <span><%= @statistics_announcement.display_date %></span>
+        <span>(<%= @statistics_announcement.confirmed? ? 'confirmed' : 'provisional' %>)</span>
+      </p>
+    </div>
+  </section>
+
+  <section class="app-view-statistics-announcements-summary__section">
+    <div class="govuk-body">
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Connected document",
+        heading_level: 2,
+        font_size: "l",
+        margin_bottom: 3,
+      } %>
+      <p>
+        <% if @statistics_announcement.publication %>
+          <%= link_to("Change connected document",
+                      admin_statistics_announcement_publication_index_path(@statistics_announcement),
+                      class: "govuk-link ,govuk-!-margin-bottom-8") %>
+        <table class="govuk-table">
```

Comment:
Using a table for a single item here feels semantically wrong to me. I wonder if we can just use a 
p tag followed by a span for the link which is aligned right. Wdyt?

Reply:
Yeah, definitely take your point. But can't get your suggested approach to work. Maybe we could catch up to pair on this?